app/controllers/api/database.php

```diff
@@ -1900,8 +1900,8 @@ function createAttribute(string $collectionId, Document $attribute, Response $re
 
         $data = (\is_string($data)) ? \json_decode($data, true) : $data; // Cast to JSON array
 
-        if (empty($data)) {
-            throw new Exception('Missing payload', 400, Exception::DOCUMENT_MISSING_PAYLOAD);
+        if (empty($data) && empty($read) && empty($write)) {
```

Comment:
I am unsure about the `empty` check here. Would this allow me to set permissions to an empty array - remove all current permissions?

If not, we might need check with `isset` or `=== null` or `=== ''` or change default value when defining `->param()` to something else. Not sure, please play around to see what works the best.